scripts/functions/requirements/osx_brew

```diff
@@ -315,6 +315,10 @@ requirements_osx_brew_libs_default()
 
   # OpenSSL version depends on the version of Ruby being installed
   case "$1" in
+    (ree-1.8*)
+      brew_openssl_package="openssl"
+      ;;
+
     (ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*|ruby-1.8*)
```

Comment:
this could be: `(ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.*|ree*)`?

Reply:
Yes, it could. But I preferred to keep it separate for clarity. Don't you think its better?

Reply:
both work :) our code grows huge so I thought a bit shorter would not harm, maybe on beginning? but go ahead and merge if you like

Reply:
Will get smaller once we re-write to ruby :)